app/models/manageiq/providers/autosde/storage_manager/event_catcher/runner.rb

```diff
@@ -33,13 +33,14 @@ def queue_event(event)
 
   def event_to_hash(event, ems_id)
     {
-      :event_type => event.event_type,
-      :source     => "AUTOSDE",
-      :ems_ref    => event.event_id,
-      :timestamp  => event.last_timestamp,
-      :full_data  => event.to_hash,
-      :ems_id     => ems_id,
-      :message    => event.description
+      :event_type               => event.event_type,
+      :source                   => "AUTOSDE",
+      :ems_ref                  => event.event_id,
+      :physical_storage_ems_ref => event.storage_system,
```

Comment:
NOTE as we handle events for more types of inventory this might be an issue unless `event.storage_system` is always present.

Reply:
e.g. what a lot of providers do is build the "base" event hash, then depending on the event type add in keys for the related inventory but this is something we can get to later

Reply:
storage_system can't be null in AutoSDE. it's a mandatory field.
As StorageResources, Volumes, etc...